include/swift/Reflection/ReflectionContext.h

```diff
@@ -1588,7 +1612,8 @@ class ReflectionContext
 
   template <typename AsyncTaskType>
   std::pair<llvm::Optional<std::string>, AsyncTaskInfo>
-  asyncTaskInfo(StoredPointer AsyncTaskPtr) {
+  asyncTaskInfo(StoredPointer AsyncTaskPtr, unsigned ChildTaskLimit,
+                unsigned AsyncBacktraceLimit) {
```

Comment:
Tortoise-and-hare would avoid having to specify these limits here, though maybe they're desirable for other reasons?

Reply:
Looking at later code, they get used to switch off the child task and backtrace code. There could be a flag for that, mind.